include/swift/AST/ConformanceAttributes.h

```diff
@@ -28,9 +30,15 @@ struct ConformanceAttributes {
   /// The location of the "unsafe" attribute if present.
   SourceLoc unsafeLoc;
 
-  /// The location of the "@isolated" attribute if present.
-  SourceLoc isolatedLoc;
-  
+  /// The location of the "nonisolated" modifier, if present.
+  SourceLoc nonisolatedLoc;
+
+  /// The location of the '@' prior to the global actor type.
+  SourceLoc globalActorAtLoc;
+
+  /// The global actor type to which this conformance is isolated.
+  TypeExpr *globalActorType = nullptr;
```

Comment:
Maybe this ought to be a TypeLoc?

Reply:
It could. We get a TypeExpr out of the CustomAttr itself, so we can either stick that into a TypeLoc or keep it as-is. This way it's only a single pointer in ConformanceAttributes vs. the two-pointer TypeLoc, so I didn't do it.